conf/gcs/gcs_with_video.xml

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE layout SYSTEM "layout.dtd">
+
+<layout width="1362" height="1008">
+  <rows>
+    <widget NAME="map2d" size="654">
+      <papget type="message_field" display="text" x="1159" y="168">
+        <property name="scale" value="1."/>
+        <property name="field" value="WEATHER:windspeed"/>
+        <property name="format" value="%.1f Wind speed"/>
+        <property name="size" value="15."/>
+        <property name="color" value="yellow"/>
+      </papget>
+      <papget type="message_field" display="text" x="1151" y="85">
+        <property name="scale" value="1."/>
+        <property name="field" value="AIRBORNE_ANTENNA:mag_heading_deg"/>
+        <property name="format" value="%.0f Mag"/>
+        <property name="size" value="25."/>
+        <property name="color" value="#00ff00"/>
```

Comment:
With color would always add a RED and BLUE component, since on some Monochrome pure green not too visible, so something like #33ff33, as soon as PR is honored will test and enhance and make and improve PR otherwise it will never land up in master :)

Reply:
Ok!